documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+
+.Procedure
+Execute the following steps for each `Kafka.spec.cruiseControl` configuration in your Kafka cluster.
```

Comment:
Not sure about _execute_. Maybe this could be reduced to: _For each `Kafka.spec.cruiseControl` configuration in your Kafka cluster:_

Reply:
We don't have a _For each_ from some of the other Kafka config changes?

Reply:
Good observations, thanks. I updated each file to: